### PR TITLE
feat(console): support render ui template

### PR DESCRIPTION
**Pull request by leoryu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by wangpand0508 — APPROVED** [review]
lgtm

File: pkg/auth/apiserver/apiserver.go [hunk 1]
```diff
@@ -119,6 +134,39 @@ type APIServer struct {
 	GenericAPIServer *genericapiserver.GenericAPIServer
 }
 
+type ConsoleConfig struct {
+	Title   string
+	LogoDir string
+}
+
+// NewAuthOptions creates a AuthOptions object with default parameters.
+func NewConsoleConfigOptions() *ConsoleConfig {
```
Comment (wl-chen) [inline pkg/auth/apiserver/apiserver.go:143]: 是不是可以在这儿加默认值，省一个cfg那里的setDefault方法

File: pkg/auth/apiserver/apiserver.go [hunk 2]
```diff
@@ -54,14 +59,23 @@ import (
...
 const (
 	OIDCPath           = "/oidc/"
 	AuthPath           = "/auth/"
 	APIKeyPasswordPath = "/apis/auth.tkestack.io/v1/apikeys/default/password"
 
-	APIKeyPath = "/apis/auth.tkestack.io/v1/apikeys"
+	APIKeyPath           = "/apis/auth.tkestack.io/v1/apikeys"
+	DefaultTitle         = "TKEStack"
+	DefaultLogoDir       = "default"
+	htmlTmplDir          = "web/auth/templates/"
```
Comment (wl-chen) [inline pkg/auth/apiserver/apiserver.go:74]: 这里的五个常量建议大写

File: pkg/auth/apiserver/apiserver.go (hunk 1 above)
Comment (leoryu) [reply pkg/auth/apiserver/apiserver.go:143]: We already have `setupDefaultConsoleConfig`.

Comment (wl-chen) [reply pkg/auth/apiserver/apiserver.go:143]: 我意思是这里新创建config的时候，就可以把默认值填上去，setupDefaultConsoleConfig 这个方法就能省了

File: pkg/auth/apiserver/apiserver.go (hunk 2 above)
Comment (leoryu) [reply pkg/auth/apiserver/apiserver.go]: done

**Review by wl-chen — APPROVED** [review]

**Pull request merged**